packages/flutter/lib/src/semantics/semantics.dart

```diff
@@ -690,14 +691,25 @@ class SemanticsProperties extends DiagnosticableTree {
   /// TalkBack/VoiceOver will treat it as non-editable text field.
   final bool readOnly;
 
+  /// If non-null, whether the node currently is able to hold input focus.
+  ///
+  /// If [focusable] is set to false, then [focused] must not be true.
+  ///
+  /// Input focus (indicates that the node will receive keyboard events) is not
+  /// to be confused with accessibility focus. Accessibility focus is the
+  /// green/black rectangular highlight that TalkBack/VoiceOver draws around the
+  /// element it is reading, and is separate from input focus.
```

Comment:
Maybe also define what inout focus means?

Reply:
That's what the `(indicates that the node will receive keyboard events)` was supposed to be doing, I think. I reworded it.